RELEASES.md

```diff
@@ -1,3 +1,641 @@
+Version 1.58.0 (2022-01-13)
+==========================
+
+Language
+--------
+
+- [Format strings can now capture arguments simply by writing `{ident}` in the string.][90473]
+- [`*const T` pointers can now be dereferenced in const contexts.][89551]
+- [The rules for when a generic struct implements `Unsize` have been relaxed.][90417]
+
+Compiler
+--------
+
+- [Add LLVM CFI support to the Rust compiler][89652]
+- [Stabilize -Z strip as -C strip][90058]
+- [Add support for LLVM coverage mapping format versions 5 and 6][91207]
+- [Emit LLVM optimization remarks when enabled with `-Cremark`][90833]
+- [Update the minimum external LLVM to 12][90175]
+- [Add `x86_64-unknown-none` at Tier 3*][89062]
+- [Build musl dist artifacts with debuginfo enabled][90733]
+- [Fix CVE-2021-42574][90462]
+- [Don't abort compilation after giving a lint error][87337]
+- [Error messages point at the source of trait bound obligations in more places][89580]
+- [Link with default MACOSX_DEPLOYMENT_TARGET if not otherwise specified.][90499]
+
+\* Refer to Rust's [platform support page][platform-support-doc] for more
+   information on Rust's tiered platform support.
+
+Libraries
+---------
+
+- [Add #[must_use] to Rc::downgrade][89833]
+- [Add #[must_use] to expensive computations][89835]
+- [Add #[must_use] to mem/ptr functions][89839]
+- [Add #[must_use] to remaining core functions][89897]
+- [Add #[must_use] to remaining alloc functions][89899]
+- [Add #[must_use] to len and is_empty][89786]
+- [Add #[must_use] to thread::Builder][89789]
+- [Add #[must_use] to alloc functions that would leak memory][90427]
+- [Add #[must_use] to remaining std functions (A-N)][90430]
+- [Add #[must_use] to remaining std functions (O-Z)][90431]
+- [Windows: Resolve `process::Command` program without using the current directory][87704]
+- [Paths are automatically canonicalized on Windows for operations that support it][89174]
+- [Re-enable `copy[_nonoverlapping]()` debug-checks][90041]
+- [Implement `RefUnwindSafe` for `Rc<T>`][87467]
+- [Make RSplit<T, P>: Clone not require T: Clone][90117]
+- [unix: Make `std::thread::available_concurrency` support process-limited number of CPUs][89310]
```

Comment:
That's not a stable function. Is it normal to put unstable changes into the release notes?

Reply:
That's also not the name of the function anymore.

Reply:
No, I missed that this was unstable. Thanks.